tests/Stub/CustomUrlRule.php

```diff
@@ -1,25 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Yiisoft\Validator\Tests\Stub;
-
-use Yiisoft\Validator\Rule\Composite;
-use Yiisoft\Validator\Rule\HasLength;
-use Yiisoft\Validator\Rule\Required;
-use Yiisoft\Validator\Rule\Url;
-
-final class CustomUrlRule extends Composite
```

Comment:
I remember it was in use but I've checked right now and it's not. Better to restore tests with this rule.

Reply:
OK. Suggest to revert the changes and restore the tests in a separate PR.